README.md

```diff
@@ -554,10 +557,11 @@ The historical release notes may helpful for general information:
 
 ### Linker
 
-The Go Linker was written by Ken Ken Thompson. Russ led a few more overhaul in Go 1.3. Austin led a rework to the linker together with Keith, Than,
+The Go Linker was written by Ken Thompson. Russ led a few more overhaul in Go 1.3. Austin led a rework to the linker together with Keith, Than,
 Cheery and many other brilliant brains, which was landed
 in Go 1.15 and Go 1.16.
 
+- [code/6l](https://github.com/golang/go/commit/0cafb9ea3d3d34627e8f492ccafa6ba9b633a213) Rob Pike. Add compiler source from inferno-os. 2008-06-05
```

Comment:
Why this link is so special?

Reply:
this link is the first import of the complete (almost) compiler source.

Reply:
I don't know the reason, I got the idea from the source's first line comments `// Inferno utils/6a/a.h
// http://code.google.com/p/inferno-os/source/browse/utils/6a/a.h` but may be I am wrong so I remove the words to avoid confuse.